src/test/test_bech32.py

```diff
@@ -56,7 +73,7 @@ def test_segwit_address(self):
             # Encode the script and make sure the address matches
             script_buf, script_len = make_cbuffer(script_hex)
             ret, retstr = wally_addr_segwit_from_bytes(script_buf, script_len, utf8(family), 0)
-            if script_ver != 0:
+            if script_ver > 16:
```

Comment:
we don't want to accept versions that we don't know how to parse/verify in wally, see 5c770b446db0bf76aebfe3293ca4e64329beaa21